File: Programming-languages-and-laboratory/Java/Exercises/31_compito_java_2019_07_05/Furgone.java
import java.util.*;

public class Furgone extends Veicolo {
    private int posti;
    private boolean rimorchio, carico;

    public Furgone(int codice, String targa, String modello, String marca, double costo, int posti, boolean rimorchio, boolean carico) {
        super(codice, targa, modello, marca, costo);
        this.posti = posti;
        this.rimorchio = rimorchio;
        this.carico = carico;
    }

    public String toString() {
        return "furgone\t" + super.toString() + "\t-\t-\t-\t" + rimorchio + "\t" + posti + "\t" + carico;
    }
}

File: II - Linguaggi di programmazione e laboratorio/Python/Exercises/31_compito_java_20190705/31_soluzione_python/furgone.py
from veicolo import Veicolo

class Furgone(Veicolo):
    rimorchio: bool
    numero_posti: int
    vano_carico: bool

    def __init__(self, codice: int, tipo: str, targa: str, marca: str, modello: str, costo_giornaliero: float, rimorchio: bool, numero_posti: int, vano_carico: bool):
        super().__init__(codice, tipo, targa, marca, modello, costo_giornaliero)
        self.rimorchio = rimorchio
        self.numero_posti = numero_posti
        self.vano_carico = vano_carico
        
    def __str__(self):
        #rimorchio, numero di posti, vano di carico
        return super().__str__() + ", -, -, -, " + str(self.rimorchio) + ", " + str(self.numero_posti) + ", " + str(self.vano_carico)